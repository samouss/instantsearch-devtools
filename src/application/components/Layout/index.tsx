import React, { SFC } from 'react';
import './index.css';

const Layout: SFC = ({ children }) => <div styleName="Layout">{children}</div>;

export default Layout;
