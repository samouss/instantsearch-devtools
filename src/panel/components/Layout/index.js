import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

const Layout = ({ children }) => (
  <div styleName="Layout">
    {children}
  </div>
);

Layout.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Layout;
