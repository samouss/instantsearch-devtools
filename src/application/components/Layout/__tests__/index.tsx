import React from 'react';
import { shallow } from 'enzyme';
import Layout from '../index';

describe('<Layout />', () => {
  it('exepct to render', () => {
    const wrapper = shallow(<Layout>This is the children.</Layout>);

    expect(wrapper).toMatchSnapshot();
  });
});
