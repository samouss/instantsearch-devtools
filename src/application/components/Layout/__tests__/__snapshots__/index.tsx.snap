// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> exepct to render 1`] = `
<div
  styleName="Layout"
>
  This is the children.
</div>
`;
